docs/specs/openapi.yaml

```
@@ -238,44 +652,361 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/ChangeObject"
-              examples:
-                get change:
-                  value:
-                    type: sync
-                    status-code: 200
-                    status: OK
-                    result:
-                      id: 72
-                      kind: autostart
-                      summary: "Autostart service \"svc1\""
-                      status: Done
-                      tasks: 
-                        - id: 72
-                          kind: start
-                          summary: "Start service \"svc1\""
-                          status: Done,
-                          progress:
-                            label: ""
-                            done: 1
-                            total: 1
-                          spawn-time: "2024-12-22T20:05:24.855436063+08:00"
-                          ready-time: "2024-12-22T20:05:25.866100522+08:00"
-                      ready: true,
-                      spawn-time: "2024-12-22T20:05:24.855443855+08:00"
-                      ready-time: "2024-12-22T20:05:25.866103439+08:00"
+                $ref: "#/components/schemas/GetChangeByIDResponse"
+              example:
+                {
+                  "type": "sync",
+                  "status-code": 200,
+                  "status": "OK",
+                  "result": {
+                    "id": "38",
+                    "kind": "autostart",
+                    "summary": "Autostart service \"svc1\" and 1 more",
+                    "status": "Done",
+                    "tasks": [
+                      {
+                        "id": "54",
+                        "kind": "start",
+                        "summary": "Start service \"svc1\"",
+                        "status": "Done",
+                        "progress": {
+                          "label": "",
+                          "done": 1,
+                          "total": 1
+                        },
+                        "spawn-time": "2024-12-27T12:31:26.673287868+08:00",
+                        "ready-time": "2024-12-27T12:31:27.681780702+08:00"
+                      }
+                    ],
+                    "ready": true,
+                    "spawn-time": "2024-12-27T12:31:26.673297951+08:00",
+                    "ready-time": "2024-12-27T12:31:27.686371869+08:00"
+                  }
+                }
+    post:
+      summary: Perform an action on a change.
```

Suggest removing period for consistency with other summaries:
```suggestion
      summary: Perform an action on a change
```